azure/requests.yaml

```diff
@@ -14,7 +14,7 @@ releases:
   - name: cluster-operator
     version: ">= 5.0.0"
   - name: app-operator
-    version: ">= 6.4.1"
+    version: ">= 6.6.3"
```

Comment:
you can't do this. 19.0.1 is already releases, so you are requesting to bump an existing release, and this is not allowed. Please add this request in the >19.0.1 section above

Reply:
Thanks! Updated 👍🏻